instrumentation/kafka-streams/src/test/java/brave/kafka/streams/ITKafkaStreamsTracing.java

```diff
@@ -208,6 +208,37 @@ public void process(String key, String value) {
     streams.cleanUp();
   }
 
+  @Test
+  public void should_create_spans_from_stream_with_tracing_filter() throws Exception {
```

Comment:
Consider to add counter example as well, `filter=true`.

Reply:
:+1: